x-pack/plugins/security_solution/scripts/endpoint/endpoint_agent_runner/elastic_endpoint.ts

```diff
@@ -8,8 +8,13 @@
 import { userInfo } from 'os';
 import execa from 'execa';
 import nodeFetch from 'node-fetch';
-import { AGENT_POLICY_SAVED_OBJECT_TYPE } from '@kbn/fleet-plugin/common';
+import type { UpdatePackagePolicyResponse, UpdatePackagePolicy } from '@kbn/fleet-plugin/common';
+import {
+  AGENT_POLICY_SAVED_OBJECT_TYPE,
+  packagePolicyRouteService,
+} from '@kbn/fleet-plugin/common';
```

Comment:
nit: you can combine type imports with the other imports from the same module
```suggestion
import {
  AGENT_POLICY_SAVED_OBJECT_TYPE,
  packagePolicyRouteService,
  type UpdatePackagePolicyResponse, 
  type UpdatePackagePolicy 
} from '@kbn/fleet-plugin/common';
```